tests/frontier/opcodes/test_swap.py

```diff
@@ -0,0 +1,129 @@
+"""
+A State test for the set of `SWAP*` opcodes.
+Ported from: https://github.com/ethereum/tests/
+blob/develop/src/GeneralStateTestsFiller/VMTests/vmTests/swapFiller.yml.
+"""
+
+import pytest  # noqa: I001
+
+from ethereum_test_forks import Fork, Frontier, Homestead
+from ethereum_test_tools import Account, Alloc, Bytecode, Environment
+from ethereum_test_tools import Opcodes as Op
+from ethereum_test_tools import StateTestFiller, Storage, Transaction
+
+
+@pytest.mark.parametrize(
+    "swap_opcode",
+    [getattr(Op, f"SWAP{i}") for i in range(1, 17)],
+    ids=lambda op: str(op),
+)
+@pytest.mark.valid_from("Frontier")
```

Comment:
add now path to original fillers. as ported marker.
rebase as converted-ethereum-tests.txt now is removed

```
@pytest.mark.ported_from(
    [
        "https://github.com/ethereum/tests/blob/v13.3/src/",
    ],
    pr=["https://github.com/ethereum/execution-spec-tests/pull/1163"],
)
```

remove the corresponding test filler .json from static_test folder in execution-spec-repo